static/themes/default-runtime.css

```
@@ -42,9 +71,9 @@
   --color-heading: var(--color-gray-900);
 
   /* used for custom brand colors */
-  --color-fg-brand-subtle: var(--color-blue-200);
-  --color-fg-brand: var(--color-blue-700);
-  --color-fg-brand-strong: var(--color-blue-900);
+  --color-fg-brand-subtle: var(--color-primary-200);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--color-fg-brand-subtle is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)